include/spmv/jds_storage.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
```

Comment:
Should `#pragma once` be used or should this follow the convention defined in `SparseMatrix.hpp`? of using:
```
#ifndef __SPMV_SparseMatrix__
#define __SPMV_SparseMatrix__
...
#endif
```